### PR TITLE
Added internal HTTPS load balancer example with http-to-https redirect

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
Intending to add this example to this page:

https://cloud.google.com/load-balancing/docs/l7-internal/int-https-lb-tf-examples

```release-note:none
```

### CODE REVIEW

**Review by melinath — CHANGES_REQUESTED** [review]
It looks like this might be breaking the compiler somehow. Are you able to generate the docs examples and/or the providers locally?

**Comment by melinath** [comment]
(If you're able to compile everything fine locally, you could try merging master / rebasing to get the latest changes in case that helps.)

**Comment by betsy-lichtenberg** [comment]
I get these errors because the path to the files isn't valid. I thought this might work because https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/templates/terraform/examples/region_target_https_proxy_basic.tf.erb#L8-L13

```
Error: Invalid function argument
│ 
│   on int_https_lb_https_redirect.tf line 62, in resource "google_compute_region_ssl_certificate" "default":
│   62:   private_key = file("path/to/private.key")
│ 
│ Invalid value for "path" parameter: no file exists at path/to/private.key; this function works only with files that are distributed as
│ part of the configuration source code, so if this file will be created by a resource in this configuration you must instead obtain this
│ result from an attribute of that resource.
╵
╷
│ Error: Invalid function argument
│ 
│   on int_https_lb_https_redirect.tf line 63, in resource "google_compute_region_ssl_certificate" "default":
│   63:   certificate = file("path/to/certificate.crt")
```

**Comment by betsy-lichtenberg** [comment]
> (If you're able to compile everything fine locally, you could try merging master / rebasing to get the latest changes in case that helps.)

It seems to be working now. My test is successful.

**Comment by betsy-lichtenberg** [comment]
Never mind about this. I fixed the issue.

**Comment by betsy-lichtenberg** [comment]
> It looks like this might be breaking the compiler somehow. Are you able to generate the docs examples and/or the providers locally?

It's working fine now, I think

**Comment by betsy-lichtenberg** [comment]
> It looks like this might be breaking the compiler somehow. Are you able to generate the docs examples and/or the providers locally?

Fixed.

**Comment by betsy-lichtenberg** [comment]
> It looks like this might be breaking the compiler somehow. Are you able to generate the docs examples and/or the providers locally?

fixed

File: mmv1/products/compute/terraform.yaml [hunk 1]
```diff
@@ -335,6 +335,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
   RegionBackendService: !ruby/object:Overrides::Terraform::ResourceOverride
     examples:
+      - !ruby/object:Provider::Terraform::Examples
```
Comment (c2thorn) [inline mmv1/products/compute/terraform.yaml:338]: Can we move this Example towards the bottom of the examples block? The order listed here is how they will appear in the documentation, and we should generally have the basic version of the resource listed first

Comment (betsy-lichtenberg) [reply mmv1/products/compute/terraform.yaml]: Done

File: mmv1/products/compute/terraform.yaml [hunk 2]
```diff
@@ -2053,6 +2053,31 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           region_url_map_name: "regionurlmap"
           home_region_backend_service_name: "home"
           region_health_check_name: "health-check"
+      - !ruby/object:Provider::Terraform::Examples
```
Comment (c2thorn) [inline mmv1/products/compute/terraform.yaml:2056]: Do you want to add a `skip_test: true` to this example for now, and then I can remove it as a part of https://github.com/hashicorp/terraform-provider-google/issues/10202 ? 

Just so we can get the sample generated?

Comment (betsy-lichtenberg) [reply mmv1/products/compute/terraform.yaml:2056]: Done

**Review by c2thorn — APPROVED** [review]

**Pull request merged**